### PR TITLE
#32 Feature: create operator and set permissions

### PR DESCRIPTION
**Pull request by Niyonkuru1**
### What does this PR do?
- CRUD operations for operator
- Setting them a role
### Description of Task to be completed?
As an admin, I want to create, update, and delete operators and set them a roles so that they perform the task they are assigned to only.

- The admin should be able to create, update, and delete a operator.
- The admin should be able to view a paginated list of operators in the system.

- Skeleton UIs/screen are/is shown when the page loads/is loading.
 - The page is responsive.
- Appropriate error messages are displayed on unsuccessful attempts
### How should this be manually tested?
`Clone the repo`

`checkout` to branch `ft-create-operator-and-set-permissions-32
`

pull the changes by `git pull`

install all packages by `running npm install`

start app by running `npm run start`

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[Trello Board](https://trello.com/c/OlZ7LyxG/32-create-operator-set-permissions)

Screenshots (if appropriate)

### CODE REVIEW

**Comment by Niyonkuru1** [comment]
### What does this PR do?
- CRUD operations for operator
- Setting them a role
### Description of Task to be completed?
As an admin, I want to create, update, and delete operators and set them a roles so that they perform the task they are assigned to only.

- The admin should be able to create, update, and delete a operator.
- The admin should be able to view a paginated list of operators in the system.

- Skeleton UIs/screen are/is shown when the page loads/is loading.
 - The page is responsive.
- Appropriate error messages are displayed on unsuccessful attempts
### How should this be manually tested?
`Clone the repo`

`checkout` to branch `ft-create-operator-and-set-permissions-32
`

pull the changes by `git pull`

install all packages by `running npm install`

start app by running `npm run start`

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[Trello Board](https://trello.com/c/OlZ7LyxG/32-create-operator-set-permissions)

Screenshots (if appropriate)

**Review by tysepa — APPROVED** [review]

File: src/dashboard/Admin.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// import React from "react";
+// import TableRow from "./TableRow";
+// import { MdArrowDropDown } from "react-icons/md";
+// import { Link } from "react-router-dom";
+// import {
```
Comment (PrinceNiyonshuti) [inline src/dashboard/Admin.js]: remember to remove the unused codes

File: src/dashboard/ErrorMessage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// import React from 'react';
+// import "../main.css"
+// import { ImSpinner9 } from "react-icons/im";
+
+// const ErrorMessage = () => {
```
Comment (PrinceNiyonshuti) [inline src/dashboard/ErrorMessage.js]: the same here with unused codes

File: src/dashboard/FormModal.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
+// import React from 'react'
+
+// const FormModal = () => {
+//   return (
```
Comment (PrinceNiyonshuti) [inline src/dashboard/FormModal.js]: same here and the rest of commented or unused codes

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
please remove the unused codes that are commented

**Review by Xldivin — APPROVED** [review]

**Review by tysepa — APPROVED** [review]
working good but correct those conflict

**Review by PrinceNiyonshuti — APPROVED** [review]
looks good

File: src/Routes/LandingPage.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  //        items-center sm:justify-between
+  //        ">
+	// 					<img
+	// 						className="w-full sm:w-1/2"
+	// 						src={landing}
+	// 						alt=""
+	// 						role="image"
+	// 					/>
+	// 					<div>
+	// 						<h1 className="text-textBluePhant font-bold leading-9 text-3xl sm:text-4xl font-Montserrat">
```
Comment (thepatrickniyo) [inline src/Routes/LandingPage.js]: Remember to remove comments for the best code quality

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
Remove comments every where in your codebase

**Review by Hodal66 — APPROVED** [review]

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**